src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+import classNames from 'classnames';
+import noop from 'lodash/noop';
+import * as React from 'react';
+import ActivityError from '../common/activity-error';
+import ActivityMessage from '../common/activity-message';
+import ActivityTimestamp from '../common/activity-timestamp';
+import AnnotationActivityLink from './AnnotationActivityLink';
+import AnnotationActivityMenu from './AnnotationActivityMenu';
+import Avatar from '../Avatar';
+import Media from '../../../../components/media';
+import messages from './messages';
+import type { ActionItemError, AnnotationActivityItem, BoxAnnotationPermission } from '../../../../common/types/feed';
```

Comment:
Types should be imported after regular modules but before SCSS files, I believe.